lib/ui/painting/canvas.cc

```diff
@@ -425,9 +406,11 @@ void Canvas::drawPath(const CanvasPath* path,
 void Canvas::drawImage(const CanvasImage* image,
                        double x,
                        double y,
-                       const Paint& paint,
-                       const PaintData& paint_data,
+                       Dart_Handle paint_objects,
```

Comment:
If this new path is fast enough we might want to get rid of Paint serizliation/deserialization like this.

Reply:
Is that something you'd like me to investigate in this PR, or should I file an issue?

Reply:
Issue is fine